src/splashScreen/spectronTest/splashScreen.spectron.test.js

```diff
@@ -0,0 +1,57 @@
+import {
```

Comment:
From the testing point of view, this is one of the most interesting files in this PR. Implemented using spectron and webdriverio under the hood we can interact with the app, take screenshots, wait for certain conditions and simulate scenarios thanks to the access to the reducer we've configured as part of the window. This last detail is quite hacky but useful 👍